contrib/agent/src/main/java/io/opencensus/contrib/agent/bootstrap/TraceStrategy.java

```diff
@@ -36,4 +37,13 @@
    */
   @MustBeClosed
   Closeable startScopedSpan(String spanName);
```

Comment:
I would suggest to use Scope instead of Closeable to avoid having to always catch exception.

Reply:
Good comments. I think we should actually have a NonThrowingCloseable to simplify the code, what do you think?

Reply:
Added Javadoc for that. Classes in the .bootstrap package are loaded by the bootstrap classloader and thus cannot directly see the classes loaded by the system classloader.